src/meshlabplugins/io_u3d/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(TARGET external-IDTFConverter)
 	set(SOURCES io_u3d.cpp)
 
 	set(HEADERS io_u3d.h)
-	include_directories("${U3D_DIR}/src/IDTF")
+	include_directories("${EXTERNAL_DIR}/u3d/src/IDTF")
```

Comment:
So these should be equivalent - was one not working?

Reply:
Yep, the first one was not working. I don't know why. Apparently using add_subdirectory makes not visible what's inside the subdirectory inside the plugins...

Reply:
oh, right! Variables have scope (usually directory). Ideally the build system for u3d would be simpler and updated so that it contained something like `target_include_directories(u3d PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/IDTF)` so this line would be unneeded, but that works for now!